### PR TITLE
Major package Update

### PR DESCRIPTION
**Pull request by theimo1221**
Following some npm audits, it might be a good idea to update some of stuns packages.

I made sure all my changes resulted in valid unit test completion.

### CODE REVIEW

**Comment by theimo1221** [comment]
@olleolleolle Could you take a look over this?

File: test/attributes/stun-address-attribute.js [hunk 1]
```diff
@@ -10,29 +10,15 @@ test('encode', () => {
...
   const expectedBuffer = Buffer.from([
-    0 /* Reserved */,
-    0x1 /* Family */,
-    0xf8,
-    0x24 /* Port */,
-    0xc0,
-    0xa8,
-    1,
-    2 /* IP */,
+    0 /* Reserved */, 0x1 /* Family */, 0xf8, 0x24 /* Port */, 0xc0, 0xa8, 1, 2 /* IP */,
```
Comment (olleolleolle) [inline test/attributes/stun-address-attribute.js]: This is a linter removing the line comments, making it difficult for a reader to scan.

Can you find ways to build up the array (perhaps in more steps, perf does not matter in this test, but legibility will matter) that the linter will accept, and that lets the code keep the by-line comments?

File: test/attributes/stun-address-attribute.js [hunk 2]
```diff
@@ -10,29 +10,15 @@ test('encode', () => {
...
   const packet = Buffer.from([
-    0 /* Reserved */,
-    0x1 /* Family */,
-    0xf8,
-    0x24 /* Port */,
-    0xc0,
-    0xa8,
-    1,
-    2 /* IP */,
+    0 /* Reserved */, 0x1 /* Family */, 0xf8, 0x24 /* Port */, 0xc0, 0xa8, 1, 2 /* IP */,
```
Comment (olleolleolle) [inline test/attributes/stun-address-attribute.js]: (Same here)

File: test/message/decode.js [hunk 3]
```diff
@@ -6,39 +6,11 @@ const { messageType } = require('lib/constants');
...
-    0x70,
-    0x7b,
-    0xb8,
-    0xcc,
-    0x6a,
-    0x63,
-    0x3a,
-    0x9d,
-    0xf7 /* Transaction */,
+    0, 0x01 /* Type */, 0, 12 /* Length */, 0x21, 0x12, 0xa4, 0x42 /* Cookie */, 0xd0, 0x05, 0x58,
```
Comment (olleolleolle) [inline test/message/decode.js]: (Same here in this file)

File: test/message/encode.js [hunk 4]
```diff
@@ -14,39 +14,11 @@ test('should encode request', () => {
...
-    0x70,
-    0x7b,
-    0xb8,
-    0xcc,
-    0x6a,
-    0x63,
-    0x3a,
-    0x9d,
-    0xf7 /* Transaction */,
+    0, 0x01 /* Type */, 0, 12 /* Length */, 0x21, 0x12, 0xa4, 0x42 /* Cookie */, 0xd0, 0x05, 0x58,
```
Comment (olleolleolle) [inline test/message/encode.js]: (Same here in this file)

**Review by olleolleolle — CHANGES_REQUESTED** [review]
Super-change! Thanks for taking the time to update the dependencies, that means a lot.

The linter did a good job of making uniform all the files.

The tests have carefully-composed arrays used as Buffer.from arguments, and it would be cool to build them in such a way that the line comments can be kept.

Perhaps like this, perhaps some other way, that lets the text remain on the same line as the value that it talks about.

```js
let input = []
input.push(0x00) // A null there.
```

**Comment by theimo1221** [comment]
@olleolleolle Thanks for the quick review! Following your suggestion I updated the PR.

**Comment by olleolleolle** [comment]
(looks good to me!)

**Comment by theimo1221** [comment]
> (looks good to me!)

I thought you might have the necessary permissions to menge this PR and run a release...

Do you know how to reach one of the owners?

**Comment by theimo1221** [comment]
@reklatsmasters Could you look into this please

**Comment by reklatsmasters** [comment]
A very controversial change, only brackets and bytes. Rejected.

**Comment by theimo1221** [comment]
Hello @reklatsmasters,

I'm sorry but this PR doesn't just include brackets and bytes.
The main purpose are those package updates and afterwards the correctness in regards to the defined linting rules.

@olleolleolle am I missing something here?

**Pull request closed without merging**